isort/deprecated/finders.py

```diff
@@ -232,7 +233,6 @@ def _load_mapping() -> Optional[Dict[str, str]]:
                 import_name, _, pypi_name = line.strip().partition(":")
                 mappings[pypi_name] = import_name
             return mappings
```

Comment:
Added in https://github.com/PyCQA/isort/commit/52c86da4f087eda2c3b65b174372ca20e50e8038 and indeed seems unnecessary